app/grandchallenge/archives/tasks.py

```diff
@@ -139,18 +118,41 @@ def update_archive_item_update_kwargs(
             civ = interface.create_instance(value=value)
             civ_pks_to_add.add(civ.pk)
 
-    return civ_pks_to_add, civ_pks_to_remove, upload_pks
+    return civ_pks_to_add, upload_pks
```

Comment:
We shouldn't return multiple values from a method. If needed, can you change this to a named tuple and update any callers?

Reply:
The return value is actually never used, as the sets are updated in place. I'm not sure how I feel about that tbh. I could just return nothing here. What do you think?

Reply:
Then yes if it's not needed remove it.